### ISSUE TITLE
One last missing period.

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#177]
While double checking description in issue https://github.com/phetsims/gravity-force-lab-basics/issues/173#issuecomment-534543641

I found one missing period in an aria-live alert (a context response), please see red arrow in screenshot below. This alert (context response) occurs when the Constant Size checkbox becomes unchecked.

Please be careful adding in the period. It is important to make sure a period does not pop up in the other underlined areas of the PDOM's dynamic descriptions (see underlined content).

If this is not straightforward, I think we can handle adding this period in a maintenance release.
![last-missing-period](https://user-images.githubusercontent.com/1902116/65513950-e17d2500-deb6-11e9-81d3-7079c2b6f27a.png)

**Comment by terracoda** [comment]
And this is an issue for GFL regular, too, so this string might live in some common strings place.
![Screen Shot 2019-09-24 at 10 46 12 AM](https://user-images.githubusercontent.com/1902116/65514761-8e0bd680-deb8-11e9-9aac-baf1afadab3f.png)

**Comment by zepumph** [comment]
We got lucky here. Though that snippet was in three places in the descriptions, there was a string pattern that only applied to the alert, so I added the period there. Please review before I copy it over to the RC shas.

**Comment by terracoda** [comment]
Awesome! Looks good @zepumph. Closing issue.
![period-fixed](https://user-images.githubusercontent.com/1902116/65543573-faec9400-deeb-11e9-9b96-cfaac6d99556.png)
